content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -211,7 +211,7 @@ PID   USER     TIME  COMMAND
 
 {{% capture whatsnext %}}
 
-* Learn how to easily [Debug a Running Pod](/docs/tasks/debug-application-cluster/debug-running-pod.md)
+* Learn how to easily [debug a running Pod](/docs/tasks/debug-application-cluster/debug-running-pod.md)
```

Comment:
nit: please fix the link by removing the `.md`,
/docs/tasks/debug-application-cluster/debug-running-pod/